code/frontend/src/App.js

```diff
@@ -28,6 +30,8 @@ function App() {
         <Route path='/' element={screenToShow()} />
         <Route path='/search' element={<Search />} />
         <Route path='/browse' element={<EducationalVideoFeed />} />
+        <Route path='/userProfile' element={<UserProfile />} />
+        <Route path='/videoHistory' element={<VideoHistory />} />
```

Comment:
NitPick: Please follow the naming convention for rest uris: https://restfulapi.net/resource-naming/

Reply:
> NitPick: Please follow the naming convention for rest uris: https://restfulapi.net/resource-naming/

changed and updated the code.